snuba/admin/static/tracing/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Client from "../api_client";
+import QueriesDisplay from "../components/query_display/query_display";
+import { QueryResult } from "../components/query_display/types";
+
+function TracingQueries(props: { api: Client }) {
+  function tablePopulator(queryResult: QueryResult) {
+    var elements = {};
+    if (queryResult.error === undefined) {
+      elements = { Trace: [queryResult.trace_output, 400] };
```

Comment:
I think we shouldn't be checking the API response code in the components like this. It's supposed to be the job of `api_client` to check the response code and reject with the right message on error

Reply:
I think Lyn is right in that error checking can happen in the QueryDisplay component. I don't think tracing and system queries should handle that differently.